src/straight_skeleton_2.h

```diff
@@ -0,0 +1,12 @@
+#ifndef COMPAS_STRAIGHT_SKELETON_2_H
+#define COMPAS_STRAIGHT_SKELETON_2_H
+
+#include <compas.h>
+
+typedef std::tuple<std::vector<double>, std::vector<double>> Edge;
+typedef std::list<Edge> Edges;
```

Comment:
perhaps we can move these guys to the compas namespace?

Reply:
same for the definitions in `skeletonization.h`...

Reply:
i put the typedefs of `Edge` and `Edges` into the compas namespace and removed them from `skeletonization` and `straight_skeleton_2`.. still failing on windows :unamused:

Reply:
It finally builds! just forgot to add the "straight_skeleton_2.cpp" to the setup.py :sweat_smile: